metrics/prometheus_fake.go

```diff
@@ -648,6 +648,22 @@ func (p testSubcontainersInfoProvider) SubcontainersInfo(string, *info.Container
 							Cpu:          1,
 						},
 					},
+					PerfUncoreStats: []info.PerfUncoreStat{
+						{
+							ScalingRatio: 1.0,
+							Value:        1231231512.0,
+							Name:         "uncore_imc_0/cas_count_read",
```

Comment:
While i'm alright with having the event name be PMU/Name for the configuration, i'd like us to separate them out for reporting of metrics. IIUC, based on prometheus metrics below, this name includes the PMU prefix.

Reply:
It's separated. Sorry, I missed change this test case.

In a real scenario prometheus metric will look like:
``container_perf_uncore_events_total{event="cas_count_read",id="/",pmu="uncore_imc_0",socket="0"} 2.106316711e+10 1592929424570``